src/pages/404.tsx

```diff
@@ -1,28 +1,22 @@
-import { Link } from 'gatsby'
-import styled, { ThemeProvider } from 'styled-components'
+import { Link, PageProps } from 'gatsby'
+import styled from 'styled-components'
 
-import HeroImage from '../components/Hero/HeroImage'
-import HeroLogo from '../components/Hero/HeroLogo'
-import NavigationMenu from '../components/NavigationMenu'
+import Page from '../components/customPageComponents/Page'
 import PageSectionContainer from '../components/PageSectionContainer'
 import TextSnippet from '../components/TextSnippet'
-import GlobalStyle from '../styles/global-style'
-import { darkTheme } from '../styles/themes'
-
-const NotFoundPage = () => (
-  <ThemeProvider theme={darkTheme}>
-    <GlobalStyle />
-    <main style={{ height: '100vh' }}>
-      <HeroImage layer="back" slide={0} parallaxSpeed={12} />
-      <HeroImage layer="middle" slide={0} parallaxSpeed={8} />
-      <HeroImage layer="front" slide={0} parallaxSpeed={2} />
-      <PageSectionContainer>
-        <div className="navigation-menu-wrapper">
-          <NavigationMenu />
-        </div>
+
+const NotFoundPage = (props: PageProps) => (
+  <Page
+    {...props}
+    seo={{
+      title: 'Alephium | The Web3 you were promised',
+      description:
+        'Alephium is the next generation PoW Layer 1 with smart contracts. Built for speed, security, and sustainability. Start building or join the community today.'
+    }}
```

Comment:
I believe this can be omitted since the default one will be used.

Reply:
Done

Reply:
Done